WIDGETS_README.md

```diff
@@ -0,0 +1,28 @@
+# Uniswap Widgets
+
+This package provides React components that wrap simple Uniswap-related functionality.
+
+## Swap Widget
+
```

Comment:
The "requires integrators" section should be a new sentence or a new paragraph, not in the same list of the features.